docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -0,0 +1,101 @@
+
+Introduction
+------------
+The Lightly Docker can be used with the Lightly Platform to do
+the following workloads in one single run:
+
+- stream your files directly from your AWS S3 bucket to your local machine without
+  needing to sync or download them
+- embed all images or video frames
+- sample a subset, e.g. using CORESET
+- compute the metadata of the images
+- create a dataset in the Lightly Platform from the sampled subset
+
+Advantages
+----------
+
+- You can run the whole Lightly workflow with one single command.
+- You can process videos directly without needing to extract frames.
+- Your data is streamed from your S3 bucket on the go instead of being first downloaded
+  to you local disk and then read from there. Thus you save both a lot of time and
+  disk space and can process much larger datasets.
+- You can :ref:`ref-docker-with-datasource-datapool`
+  to continuously update and improve your subsampled dataset
+  everytime new data comes into your S3 bucket.
+
+
+Requirements
+------------
+
+This recipe requires that you already have a dataset in the Lightly Platform
+configured to use the data in your AWS S3 bucket.
+
+Follow the steps on how to `create a Lightly dataset connected to your S3 bucket <https://docs.lightly.ai/getting_started/dataset_creation/dataset_creation_aws_bucket.html>`_.
+
+Furthermore, you should have access to a machine running docker.
+Ideally, it also has a CUDA-GPU.
+A fast GPU will speed up the process significantly,
+especially for large datasets.
+
+
+Download the Lightly Docker
+---------------------------------------------
+Next, the Lightly Docker should be installed.
+Please follow the instructions for the :ref:`ref-docker-setup`.
+
+
+Run the Lightly Docker with the datasource
+------------------------------------------
+Head to the :ref:`rst-docker-first-steps` to get a general idea of what the docker
+can do.
+
+For running the docker with a remote datasouce,
+use the parameter `datasource.dataset_id=YOUR_DATASET_ID`.
+You find the dataset id in the Lightly Platform.
+E.g. run the docker with
+
+.. code-block:: console
+
+    docker run --gpus all --rm -it \
+        -v OUTPUT_DIR:/home/output_dir \
+        lightly/sampling:latest \
+        token=YOUR_LIGHTLY_PLATFORM_TOKEN \
+        datasource.dataset_id=YOUR_DATASET_ID
+
+View the progress of the Lightly Docker
+---------------------------------------
+
+To see the progress of your docker run, go to the Lightly Platform and
+head to `My Docker Runs <https://app.lightly.ai/docker/runs>`_
+
+.. image:: ../getting_started/images/docker_runs_overview.png
+
+Use your subsampled dataset
+---------------------------
+
+Once the docker run has finished, you can use your subsampled dataset as you like:
+E.g. you can analyze it in the embedding and metadata view of the Lightly Platform,
+subsample it further, or export it for labeling.
+
+.. _ref-docker-with-datasource-datapool:
+Process new data in your S3 bucket using a datapool
```

Comment:
This is sort of the killer feature that appears in the end and it goes under because it's just lots of text. I would try to use bullet points or images/ screenshots to help. E.g. 

Running the docker for the first time:
![image](https://user-images.githubusercontent.com/1522766/150112072-9aa59ef3-c025-4ad5-b3d4-ca47a597d9ad.png)

Running the docker again with a new video (we see a new cluster because the new video is very different):
![image](https://user-images.githubusercontent.com/1522766/150112145-c748c5be-f115-407c-8529-1e4c898c48c6.png)

Only visualizing the newly added data:
![image](https://user-images.githubusercontent.com/1522766/150112263-96bad7ba-5408-447e-a2e8-310f3385cac8.png)

Reply:
I added much more content and screenshots.